hack/release/common.sh

```diff
@@ -134,6 +134,8 @@ publishHelmChart() {
     helm package "${CHART_NAME}" --version "$HELM_CHART_VERSION"
     helm push "${HELM_CHART_FILE_NAME}" "oci://${RELEASE_REPO}"
     rm "${HELM_CHART_FILE_NAME}"
+    digest="$(crane digest "${RELEASE_REPO}:${HELM_CHART_VERSION}")"
```

Comment:
nit: Is it worth creating a separate function to `cosignChart()` similar to how we have a separate function right now for `cosignImages()`

Reply:
Sure, I'll do that.

Reply:
Done